include/vfn/nvme/ctrl.h

```diff
@@ -45,6 +45,11 @@ struct nvme_ctrl {
 	 */
 	struct vfio_pci_device pci;
 
+	/**
+	 * @serial: Serial number from controller
+	 */
+	char serial[20];
```

Comment:
This bloats `struct nvme_ctrl` and it does not seem to be used in this PR.

Reply:
I thought I had put something in the commit message. It's for enumeration in xNVMe. MacVFN reads this serial and applies it to its name in macOS's "IO registry" of devices.

Reply:
OK, but it belongs in MacVFN then.